### PR TITLE
Feed to alliance side

### PR DESCRIPTION
**Pull request by uncookednehuel**
Added button to feed note to alliance side

### CODE REVIEW

**Review by tsieben — APPROVED** [review]

File: src/main/java/frc/robot/commands/VarFeedPrime.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  private Elevator elevator;
+  private PoseEstimator poseEstimator;
+  private Translation2d plopTranslation;
+  // private NetworkTableEntry lInput;
+
+  // Translation of where the note exits in the XZ plane (side view)
+  private final Translation2d shooterXZTrans;
+
+  private final DoubleSupplier wristAngle; // Range of velocity from min distance to max distance
+  private final double RPM_MPS;
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/VarFeedPrime.java:33]: Save this as `MPS_TO_RPM` and set as `1 / RPM_MPS` that you get in the constructor

File: src/main/java/frc/robot/commands/VarFeedPrime.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  private Shooter shooter;
+  private Elevator elevator;
+  private PoseEstimator poseEstimator;
+  private Translation2d plopTranslation;
+  // private NetworkTableEntry lInput;
+
+  // Translation of where the note exits in the XZ plane (side view)
+  private final Translation2d shooterXZTrans;
+
+  private final DoubleSupplier wristAngle; // Range of velocity from min distance to max distance
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/VarFeedPrime.java:32]: This comment ain't correct

File: src/main/java/frc/robot/commands/VarFeedPrime.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  @Override
+  public void execute() {
+    Pose2d pose = poseEstimator.getFusedPose();
+
+    // Distance and height to speaker
+    double l = pose.getTranslation().getDistance(plopTranslation) - shooterXZTrans.getX();
+    // double l = lInput.getDouble(3);
+
+    double h = shooterXZTrans.getY() - Units.inchesToMeters(elevator.getElevatorPos());
+    h = -h;
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/VarFeedPrime.java:75]: Can you give these variables better names? And comment why we invert the height

**Review by ArchdukeTim — CHANGES_REQUESTED** [review]

**Comment by ArchdukeTim** [comment]
This still doesn't aim at the amp!

**Review by ArchdukeTim — APPROVED** [review]

**Pull request merged**